app/views/unattended/report_templates/host_-_applied_errata.erb

```diff
@@ -46,11 +54,12 @@ require:
   version: 3.12.0
 -%>
 <%- report_headers 'date', 'hostname', 'erratum_id', 'erratum_type', 'status' -%>
-<%- load_errata_applications(filter_errata_type: input('Filter Errata Type'), 
+<%- load_errata_applications(filter_errata_type: input('Filter Errata Type'),
                              include_last_reboot: input('Include Last Reboot'),
                              since: input('Since'),
                              up_to: input('Up to'),
                              status: input('Status')
```

Comment:
missing comma causes syntax error

Reply:
:facepalm: although here is an answer to "What is RuboCop good for?" ^_^
"Find developer's stupid mistakes before it even gets to the reviewer"